libs/mrwebrtc/include/audio_track_source_interop.h

```diff
@@ -7,20 +7,6 @@
 
 extern "C" {
```

Comment:
This file is probably worth deleting now.

Reply:
I actually started implementing external audio sources, which is why a common base class is needed. But there's a fundamental issue with that, so I can't submit that change. So in the meantime I wanted to merge at least the prerequisite structural change to make external audio sources an incremental non-breaking change. Also I see no reason why we can't enable local audio callbacks on sources and that would go there.